src/utils/platform.js

```diff
@@ -0,0 +1,3 @@
+import { Platform } from 'react-native';
+
+export const CLOUD_PLATFORM = Platform.OS === 'ios' ? 'iCloud' : 'Google Drive';
```

Comment:
I think it should be more "cloudPlatform". It's not constant i thik. But i don't have a strong opinion